src/biz/tugay/groovyship/cli/CliGameController.groovy

```diff
@@ -0,0 +1,107 @@
+package biz.tugay.groovyship.cli
+
+import biz.tugay.groovyship.modal.Board
+import biz.tugay.groovyship.service.GameService
+
+import java.nio.charset.StandardCharsets
+
+import static java.lang.Integer.parseInt
+
+class CliGameController
+{
+
+  static GameService gameService = new GameService()
+
+  static Scanner scanner = new Scanner(System.in, StandardCharsets.UTF_8.toString())
+
+  static BoardCommandLinePrinter boardCommandLinePrinter = new BoardCommandLinePrinter()
+
+  static Board board = null
+
+  static void main(String[] args) {
+    println "Type ng to create a new game at any time or exit to stop playing."
+    println "Send missile at coordinates in the format: columnIndex,rowIndex."
+    println "Top left corner is coordinates 0,0"
+    println "Type ng to start a new game now."
+
+    while (true) {
+      if (board) {
```

Comment:
See the Groovy-truth to understand how this works: https://groovy-lang.org/semantics.html#Groovy-Truth

`null`, empty list, 0 is considered to be `false` in Groovy..

Reply:
Similar to python: 

empty_str = '' # or = None 
if empty_str:
   print(f'empty_str is True. Thus the str has content: {empty_str})
elif not empty_str:
  print('not empty_str is True. Thus the str is empty!')